mcs_benchmark_data/models/benchmark_dev_dataset.py

```diff
@@ -12,4 +12,10 @@
 class BenchmarkDevDataset(BenchmarkDataset):
     """A dataset containing dev samples of a benchmark"""
 
-    type: "BenchmarkDevDataset"
+    type: str
```

Comment:
Here you can do
`type: str = BenchmarkDevDataset`

Reply:
But you don't really need that `type` field at all, since it's the same for every instance of this class. You just need to serialize the constant value in to_rdf.